ribbon/views.py

```diff
@@ -0,0 +1,12 @@
+import subprocess
+from django.shortcuts import render
+from giturlparse import parse
+
+
+def index(request):
+    git_remote = (subprocess.check_output(
+        ['git', 'ls-remote', '--get-url']).decode())
+    git_remote = git_remote.strip()[:-4]
```

Comment:
What you need here should already be in `community/git.py`

Reply:
okk